### ISSUE TITLE
[Game]: Warframe

### ISSUE THREAD

**Issue by hunter-chief** [ItsLogic/Xbox-Achievement-Unlocker#281]
### Confirmation

- [X] I have verified that this is not a duplicate request.

### Game Link

https://www.trueachievements.com/game/Warframe/achievements

### Game Platform

Console and PC

### Additional Game Information

the achievements are continuously repetitive.

**Comment by Exohayvan** [comment]
I second this, would love this game to be done.

**Comment by Z13TY** [comment]
Yes please, so many buggy acheivements and a lot dont transfer over to xbox.
